src/types/Pages.tsx

```diff
@@ -0,0 +1,4 @@
+export enum Pages {
+  Home = 'Home',
+  PEOPLE = 'People',
```

Comment:
enum's elemetns should be in camelCase 
```suggestion
  People = 'People',
```